app/views/accounts/edit.html.slim

```diff
@@ -3,19 +3,22 @@ p
   |User id: 
   = @accounts.try(:id)
 
-= form_for(@accounts) do |f|
-   
-    = f.label :email 
+= form_for(@accounts) do |f| 
+    = f.label :email, 'Email: '
     = f.text_field :email 
     br
-    = f.label :password 
+    = f.label :password, 'Password: '
     = f.text_field :password, required: false 
     br
-    = f.label :first_name 
+    = f.label :first_name, 'First name: '
     = f.text_field :first_name, required: false
     br
-    = f.label :lasr_name 
+    = f.label :lasr_name, 'Last name: '
     = f.text_field :last_name, required: false
     br
+     = f.check_box :admin
+     = f.label :admin, 'Admin'
+     br
     = f.submit
-=link_to 'Delete user account', accounts_path(@account), method: :delete, data: { confirm: "Delete this user?" } 
\ No newline at end of file
+
+=link_to 'Delete user account', accounts_path(@accounts), method: :delete, data: { confirm: "Delete this user?" } 
```

Comment:
You are using locals so you should not use `@accounts`. Please check all views to avoid such issues